tutorials/interoperability/remote-staking-xcm.md

```diff
@@ -45,11 +45,13 @@ _2022年12月14日 | 作者：Kevin Neilson_
 
 该脚本将返回32字节和20字节的地址。此处，我们感兴趣的是以太坊格式的帐户，即20字节的帐户。请随时在[Moonscan](https://moonbase.moonscan.io/){target=blank}上查找您的多地点衍生账户。您会注意到此帐户是空的。您需要至少用1.1个DEV为该帐户注入资金。由于这是水龙头分配的量，您需要至少发起两次水龙头请求，或者您可以随时通过[Discord](https://discord.com/invite/amTRXQ9ZpW){target=blank}联系我们请求更多的DEV Token。
 
+![Creating the Multilocation Derivative Account](/images/tutorials/interoperability/remote-staking-via-xcm/xcm-stake-1.png)
+
```

Comment:
```suggestion
```

I think we should hold off on this screenshot til we do the translations. Deleting it for now